### PR TITLE
Add cobertura goal back

### PR DESCRIPTION
**Pull request by donovanmuller**
Not sure if you explicitly need it?
Perhaps investgate further at a later date.

### CODE REVIEW

File: nucleus/src/test/java/za/co/absa/subatomic/GluonApplicationTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GluonApplicationTest {
+
+    @Autowired
+    private NucleusConfigurarion nucleusConfigurarion;
```
Comment (kbristow) [inline nucleus/src/test/java/za/co/absa/subatomic/GluonApplicationTest.java]: Spelling

File: nucleus/src/main/java/za/co/absa/subatomic/infrastructure/NucleusConfigurarion.java [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 import org.springframework.http.client.BufferingClientHttpRequestFactory;
 import org.springframework.http.client.SimpleClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
-import org.springframework.web.filter.CommonsRequestLoggingFilter;
 
 import java.util.Collections;
 
 @Configuration
+@EnableConfigurationProperties(AtomistConfigurationProperties.class)
 public class NucleusConfigurarion {
```
Comment (kbristow) [inline nucleus/src/main/java/za/co/absa/subatomic/infrastructure/NucleusConfigurarion.java]: Spelling

File: nucleus/src/test/java/za/co/absa/subatomic/GluonApplicationTest.java (hunk 1 above)
Comment (donovanmuller) [reply nucleus/src/test/java/za/co/absa/subatomic/GluonApplicationTest.java]: Arg! 💩

**Comment by kbristow** [comment]
Change the spelling of "NucleusConfigurarion" - see comments

Add references to issues this fixes (Unless I am blind and they are here somewhere)

**Comment by donovanmuller** [comment]
@kbristow you're blind 👀 

screen shot 2018-02-14 at 12 59 44

**Pull request merged**